tools/nimgrep.nim

```diff
@@ -33,13 +33,21 @@ Options:
   --word, -w          the match should have word boundaries (buggy for pegs!)
   --ignoreCase, -i    be case insensitive
   --ignoreStyle, -y   be style insensitive
+  --nWorkers:N, -n:N  speed up search by N additional workers (threads)
```

Comment:
instead: `--threads:N, -t:N  speed up search by N additional threads`

Reply:
I chose --threads and -j, as in ripgrep